config/helm/aws-node-termination-handler/README.md

```diff
@@ -69,6 +69,7 @@ The configuration in this table applies to all AWS Node Termination Handler mode
 | `nodeSelector`                     | Expressions to select a node by it's labels for pod assignment. In IMDS mode this has a higher priority than `daemonsetNodeSelector` (for backwards compatibility) but shouldn't be used.                                                                                                                                                                                              | `{}`                                                  |
 | `affinity`                         | Affinity settings for pod assignment. In IMDS mode this has a higher priority than `daemonsetAffinity` (for backwards compatibility) but shouldn't be used.                                                                                                                                                                                                                            | `{}`                                                  |
 | `tolerations`                      | Tolerations for pod assignment. In IMDS mode this has a higher priority than `daemonsetTolerations` (for backwards compatibility) but shouldn't be used.                                                                                                                                                                                                                               | `[]`                                                  |
+| `topologySpreadConstraints`        | [Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) for pod scheduling. Useful with a highly available deployment to reduce the risk of running multiple replicas on the same Node                                                                                                                                  | `[]`                                                  |
```

Comment:
Since the Deployment is only created in Queue Processor mode please move this to the section for those values (starts on line 99).

Reply:
Updated! Thanks for pointing it out.